### DISCUSSION TITLE
Possibility for tray icons on multiple taskbards for multiple screens

### DISCUSSION THREAD

**Discussion by B100dh4nd** [valinet/ExplorerPatcher#3295] [Ideas]
Hello, I was wondering if there's a possibility to add an option to have the tray icon be on multiple screens? So one can access the tray icons without having to minimize a program that is in fullscreen?
Or if someone could explain to me how to do that if it's already a feature. Thanks!
![TX9cElLOK1](https://github.com/valinet/ExplorerPatcher/assets/105532554/7f588774-e486-42ff-8b52-45a6159677c4)

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

**Comment by Torpedro1978** [comment]
This is a feature request, why would you need a version for this?

Explorer Patcher
Version 22621.3527.65.5

Wlicrosoft Windows
Version 23H2 (0S Build 22631.4037)

I would also like to  have the tray on all taskbars / all Monitors.
